pom.xml

```diff
@@ -37,34 +37,28 @@
   </pluginRepositories>
 
   <properties>
+    <revision>2.32.0</revision>
+    <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.289.3</jenkins.version>
-    <java.level>8</java.level>
-    <artifactory-maven-plugin.version>2.6.1</artifactory-maven-plugin.version>
-    <project.build.testSourceDirectory>src/test/java</project.build.testSourceDirectory>
+    <jenkins.version>2.387.3</jenkins.version>
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.289.x</artifactId>
-        <version>1181.v04b_21d4b_0d6c</version>
+        <artifactId>bom-2.387.x</artifactId>
+        <version>2423.vce598171d115</version>
         <scope>import</scope>
         <type>pom</type>
       </dependency>
-      <dependency>
-        <groupId>org.apache.commons</groupId>
-        <artifactId>commons-lang3</artifactId>
-        <version>3.13.0</version>
-      </dependency>
     </dependencies>
   </dependencyManagement>
 
   <dependencies>
     <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-cps-global-lib</artifactId>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>pipeline-groovy-lib</artifactId>
```

Comment:
This plugin was split into two plugins, one deprecated and one non-deprecated. Switching this plugin's dependency to the non-deprecated one.